docs/development_testing.md

```diff
@@ -18,7 +18,7 @@ Running Tox will run unit tests against the locally available Pythons, as well a
 
 Tox uses virtual environments under the hood to create isolated testing environments. The tox virtual environments will get out-of-date when requirements change, causing test errors. Run `tox -r` to tell Tox to recreate the virtual environments.
 
-OSX users may see an `Error creating virtualenv` when runnng `tox`. If this occurs, install the [tox-venv](https://pypi.org/project/tox-venv/) package using the command `pip install tox-venv` and try again.
+macOS users may see an `Error creating virtualenv` when runnng `tox`. If this occurs, install the [tox-venv](https://pypi.org/project/tox-venv/) package using the command `pip install tox-venv` and try again.
```

Comment:
This is a start of a sentence. I think this should be 

```suggestion
MacOS users may see an `Error creating virtualenv` when runnng `tox`. If this occurs, install the [tox-venv](https://pypi.org/project/tox-venv/) package using the command `pip install tox-venv` and try again.
```

Reply:
Normally that would make sense but Apple seems to always use "macOS" even when it's at the start of a sentence. They only seem to capitalize it when it's just "Mac" by itself without the OS part. 

See the opening paragraph (and other sections) on this page for example: https://www.apple.com/macos/what-is/

Reply:
@SeanPM5 is correct.